.github/workflows/deploy.yml

```diff
@@ -14,8 +14,8 @@ jobs:
         # https://download-installer.cdn.mozilla.net/pub/firefox/releases/
         # Chrome versions
         # https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
-        firefox_version: ["91.9.1esr"]
-        chrome_version: ["103.0.5060.114"]
+        firefox_version: ["102.4.0esr"]
+        chrome_version: ["107.0.5304.87-1"]
```

Comment:
I believe you forgot to update these versions

Reply:
Fixed, thank you